package.json

```diff
@@ -17,6 +17,7 @@
   },
   "dependencies": {
     "@hookform/resolvers": "^3.3.1",
+    "@internationalized/date": "^3.5.4",
```

Comment:
It's probably preferable to inherit this from ems-framework unless it's required here?

Reply:
Would using it in in the dob component for the onChange function count as required?

Reply:
I don't think so; it's a nice-to-have, but you should get the dependency transitively from the styleguide. My worry here is that I had some issues with Yarn not resolving ^3.5.0 and ^3.5.4 to the same version in core that lead to a lot of typescript errors.

Reply:
Would you please mind guiding me through how to update the dependency? I'm not sure I know how

Reply:
What you have here is fine.